yarn.lock

```diff
@@ -4766,6 +4738,10 @@ wordwrap@~1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/wordwrap/-/wordwrap-1.0.0.tgz#27584810891456a4171c8d0226441ade90cbcaeb"
 
+wordwrap@0.0.2:
```

Comment:
What version of yarn are you running? It looks like we both keep committing changes to the lockfile that do nothing more than reorder things around lol